zh/rules/always-prepare-statement.md

```diff
@@ -73,5 +73,5 @@ $sqlQuery = "SELECT column FROM table WHERE id = 10";
 -->
 
 ## 进一步阅读
-*[准备称述](http://php.net/manual/en/mysqli.quickstart.prepared-statements.php)
+*[准备称述](http://php.net/manual/mysqli.quickstart.prepared-statements.php)
 
```

Comment:
That's weird. @David. For me a link with `manual` only will get an empty page sometimes. 




Hi, @Damien. Can you confirm which is the best?

If David confirmed the links without `manual/en` is also working in China.

I will change them.


—
Sincerely
Buck

On Sat, Jun 27, 2015 at 1:43 PM, david-adam <notifications@github.com>
wrote:

>> @@ -73,5 +73,5 @@ $sqlQuery = "SELECT column FROM table WHERE id = 10";
>>  -->
>>  
>>  ## 进一步阅读
>> -*[准备称述](http://php.net/manual/en/mysqli.quickstart.prepared-statements.php)
>> +*[准备称述](http://php.net/manual/mysqli.quickstart.prepared-statements.php)
>>  
> It's automatically redirect to zh version.
> http://php.net/manual/mysqli.quickstart.prepared-statements.php =>
> http://php.net/manual/zh/mysqli.quickstart.prepared-statements.php
> On Sat, Jun 27, 2015 at 7:09 PM, cxbig <notifications@github.com> wrote:
>> In zh/rules/always-prepare-statement.md
>> <https://github.com/dseguy/clearPHP/pull/70#discussion_r33411871>:
>>
>> > @@ -73,5 +73,5 @@ $sqlQuery = "SELECT column FROM table WHERE id = 10";
>> >  -->
>> >
>> >  ## 进一步阅读
>> > -*[准备称述](http://php.net/manual/en/mysqli.quickstart.prepared-statements.php)
>> > +*[准备称述](http://php.net/manual/mysqli.quickstart.prepared-statements.php)
>> >
>>
>> And, If you wanna remove the language from URL.
>> You should remove manual/en/. Not only en/
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/dseguy/clearPHP/pull/70/files#r33411871>.
>>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/dseguy/clearPHP/pull/70/files#r33412141